README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><a href="https://nodei.co/npm/discord-anti-spam/"><img src="https://nodei.co/npm/discord-anti-spam.png"></a></p>
+# THIS IS THE DEV BRANCH! It's been used for development, DO NOT USE THIS OPTIONS FOR MAIN USE!!!!!!!
```

Comment:
```suggestion
<p align="center"><a href="https://nodei.co/npm/discord-anti-spam/"><img src="https://nodei.co/npm/discord-anti-spam.png"></a></p>
```

You should not change this text, it will complicate the merge into the master branch once we will want to release the updae